Index: Engine/RXStack.h
===================================================================
--- Engine/RXStack.h	(revision 85)
+++ Engine/RXStack.h	(working copy)
@@ -44,4 +44,6 @@
 - (NSString *)key;
 - (NSThread *)thread;
 
+- (NSString *)description;
+
 @end
Index: Engine/RXCardDescriptor.h
===================================================================
--- Engine/RXCardDescriptor.h	(revision 85)
+++ Engine/RXCardDescriptor.h	(working copy)
@@ -25,4 +25,6 @@
 
 - (id)initWithStack:(RXStack *)stack ID:(unsigned short)ID;
 
+- (NSString *)description;
+
 @end
Index: Engine/RXCard.h
===================================================================
--- Engine/RXCard.h	(revision 85)
+++ Engine/RXCard.h	(working copy)
@@ -48,4 +48,6 @@
 - (void)prepareForRendering;
 - (void)startRendering;
 
+- (NSString *)description;
+
 @end
Index: Engine/RXStack.m
===================================================================
--- Engine/RXStack.m	(revision 85)
+++ Engine/RXStack.m	(working copy)
@@ -352,4 +352,8 @@
 
 #endif
 
+- (NSString *)description {
+	return [NSString stringWithFormat: @"%@{%@}", [super description], _key];
+}
+
 @end
Index: Engine/RXCardDescriptor.m
===================================================================
--- Engine/RXCardDescriptor.m	(revision 85)
+++ Engine/RXCardDescriptor.m	(working copy)
@@ -118,4 +118,8 @@
 	return YES;
 }
 
+- (NSString *)description {
+	return [NSString stringWithFormat: @"%@ %hu", [_parent key], _ID];
+}
+
 @end
Index: Engine/RXCard.m
===================================================================
--- Engine/RXCard.m	(revision 85)
+++ Engine/RXCard.m	(working copy)
@@ -899,4 +899,8 @@
 	[super dealloc];
 }
 
+- (NSString *)description {
+	return [NSString stringWithFormat: @"%@{%@}", [super description], [_descriptor description]];
+}
+
 @end
